sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.controller.service.ts

```diff
@@ -213,6 +224,8 @@ export class EducationProgramControllerService {
       effectiveEndDate: getISODateOnlyString(program.effectiveEndDate),
       assessedDate: program.assessedDate,
       assessedBy: getUserFullName(program.assessedBy),
+      isBCPublic:
```

Comment:
Can we use the methods from the institution type `isBCPrivate()` and `isBCPublic()`?

Reply:
It is updated to use institution type `isBCPrivate` and `isBCPublic`. Please have a look.